tests/qunit.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<title>QUnit Example</title>
+
+	<link rel="stylesheet" href="qunit/qunit-1.19.0.css">
+
+	<script src="../js/jquery-2.1.4.min.js"></script>
```

Comment:
I only have jquery-1.9.1.min.js in js dir in my copy of repository. I think it would make sense to update jquery version used (e.g. under fundraising dir there is already 1.11.1). But as jquery 2 is not compatible with IE 8 (yeah!) we are forced to stick to 1.x, I suppose.